importers/importer.go

```diff
@@ -0,0 +1,24 @@
+package importers
+
+import (
+	"github.com/algorand/indexer/data"
+	"github.com/algorand/indexer/plugins"
+	"github.com/sirupsen/logrus"
+)
+
+// Importer defines the interface for importer plugins
+type Importer interface {
+	// Metadata associated with each Importer.
+	Metadata() ImporterMetadata
+
+	// Init will initialize each importer with a given config. This config will contain the Unmarhsalled config file specific to this plugin.
+	// It is called during initialization of an importer plugin such as setting up network connections, file buffers etc.
+	Init(cfg plugins.PluginConfig, logger *logrus.Logger) error
+
+	// Close function is used for closing network connections, files, flushing buffers etc.
+	Close() error
+
+	// GetBlock, given any round number-rnd fetches the block at that round
+	// It returns an object of type BlockExportData defined in exporters plugin
+	GetBlock(rnd uint64) (*data.BlockData, error)
```

Comment:
Is there a reason we are returning `BlockData` as a pointer?  Is it optional (i.e. if GetBlock returns nothing).